packages/slice-machine/lib/models/common/widgets/ContentRelationship/Form.tsx

```diff
@@ -10,39 +10,44 @@ import { Col, Flex as FlexGrid } from "components/Flex";
 import { createFieldNameFromKey } from "@lib/forms";
 import { useSelector } from "react-redux";
 import { selectAllCustomTypes } from "@src/modules/availableCustomTypes";
+import { FormikProps } from "formik";
 
 const FormFields = {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
   label: DefaultFields.label,
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
   id: DefaultFields.id,
   customtypes: {
     validate: () => yup.array().of(yup.string()),
   },
 };
 
+type FormProps = {
+  config: { label: string; select: string; customtypes: string[] };
+  id: string;
+  // type: string; // TODO: this exists in the yup schema but this doesn't seem to be validated by formik
+};
+
 const WidgetForm = ({
   initialValues,
   values: formValues,
   fields,
   setFieldValue,
-}) => {
+}: FormikProps<FormProps> & { fields: Record<string, unknown> }) => {
   const customTypes = useSelector(selectAllCustomTypes);
 
   const options = customTypes.map((ct) => ({
     value: ct.local.id,
     label: ct.local.label,
   }));
 
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
   const selectValues = formValues.config.customtypes
-    ? // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-      formValues.config.customtypes.map((id) => {
-        const ct = customTypes.find(
-          (frontendCustomType) => frontendCustomType.local.id === id
-        );
-        return { value: ct?.local.id, label: ct?.local.label };
-      })
+    ? formValues.config.customtypes
+        .map((id) => {
+          const ct = customTypes.find(
+            (frontendCustomType) => frontendCustomType.local.id === id
+          );
+          return { value: ct?.local.id, label: ct?.local.label };
```

Comment:
Could you return undefined instead of the object when the ct is not found and filter out the undefined below?